<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StateMachine</class>
 <widget class="QDialog" name="StateMachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ChangeState">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change State</string>
   </property>
  </widget>
  <widget class="QLabel" name="StateLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Current State</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
